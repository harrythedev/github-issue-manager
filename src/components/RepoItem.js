import React from 'react'
import styled from 'styled-components'

const RepoItem = ({ repo, getRepoIssues }) => (
  <li>
    <div
      onClick={() => getRepoIssues({ fullName: repo.full_name, id: repo.id })}
    >
      {repo.full_name}
    </div>
  </li>
)

export default RepoItem
