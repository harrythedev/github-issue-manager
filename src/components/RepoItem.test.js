import React from 'react'
import RepoItem from './RepoItem'
import renderer from 'react-test-renderer'

it('renders ', () => {
  const repo = {
    id: 169527722,
    full_name: 'harrythedev/github-issue-manager',
  }
  const tree = renderer.create(<RepoItem repo={repo} />).toJSON()
  expect(tree).toMatchSnapshot()
})
