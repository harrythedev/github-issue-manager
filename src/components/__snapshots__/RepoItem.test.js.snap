// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders  1`] = `
<li>
  <div
    onClick={[Function]}
  >
    harrythedev/github-issue-manager
  </div>
</li>
`;
